auth/realm/token/src/main/java/org/wildfly/security/auth/realm/token/_private/ElytronMessages.java

```diff
@@ -80,6 +82,10 @@
     @Message(id = 1128, value = "Jwt-based token not configured with a public key. Ignoring signature verification.")
     void tokenRealmJwtWarnNoPublicKeyIgnoringSignatureCheck();
 
+    @LogMessage(level = WARN)
+    @Message(id = 1177, value = "Not sending new request to jwks url \"%s\". Last request time was %d.")
```

Comment:
It looks like this id is already being used elsewhere. I think you'll need to make use of a new id range. Take a look in ELY_Messages.txt.

Reply:
Ups! Sorry! I thought we had the whole range for us. I have added a new range from 1181 to 1185 (the following 5) because they are free. Modified the txt file too.